docs/pages/components/asset-upload.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Asset Upload
+id: asset-upload
+keywords: asset upload, file upload, asset, file, upload
+sidebar: left-navigation-sidebar
+toc: false
+permalink: components/asset-upload.html
+folder: components
+summary:
+---
+
+Asset Upload provides a stylized component for file upload button.
+{: .docs-intro}
+
+## Default Asset Upload
```

Comment:
It seems odd to call this the "default" when the version with no message simply omits the `fd-asset-upload__message` element.  It seems like the default should have required information (_e.g. "text") so that it would have an icon and text, but no message.  Then there should be an example with additional message and then the last option with no icon.  I'm just thinking of how this would be implemented in a functional library.  There would a "no icon" option (which deviates from the default) and then it would seem strange to have a "no message" option when all you would really need to do is just not pass the message text.

Reply:
I changed the default test to "Asset upload with Icon and Message" there are no clear guidelines on what the default should be from the design side. 

<img width="659" alt="Screen Shot 2019-07-22 at 12 12 31 PM" src="https://user-images.githubusercontent.com/15215400/61650816-21871800-ac7a-11e9-9bfc-73019e228bf9.png">